packages/wrangler/src/dev/local.tsx

```diff
@@ -197,6 +195,9 @@ function useLocalWorker({
 				bundle,
 			});
 
+			const runtimeInspectorPort = (runtimeInspectorPortRef.current ??=
```

Comment:
I know that this works, but... can it be split onto another line? Or an if statement?